widget/src/providers/ChatProvider.tsx

```diff
@@ -453,6 +453,10 @@ const ChatProvider: React.FC<{
     handleSubscription,
   };
 
+  useBroadcastChannel("session", () => {
+    socketCtx.socket.disconnect();
+  });
```

Comment:
Shouldn't we check on the message "logout" ?

Reply:
Yes, the logic is updated to check the body of sent message